extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -66,6 +66,9 @@ describes.realWin(
         .map((unused, i) => {
           const page = win.document.createElement('amp-story-page');
           page.id = opt_ids && opt_ids[i] ? opt_ids[i] : `-page-${i}`;
+          page.signals = () => ({
+            whenSignal: () => Promise.resolve(),
+          });
```

Comment:
Is this really needed? We thought this change would be a no-op, if it's breaking our tests I'd suspect it might break something in prod :(
Maybe fixing the `amp-story-page` code (cf previous comment) will fix this?

Reply:
we chatted offline and figured we should just mimic what the code does in amp-story in this test, and wait for the `page.whenSignal(LOAD_END)` to finish before the evaluation.